streaming/vision/convert/ade20k.py

```diff
@@ -154,16 +154,18 @@ def main(args: Namespace) -> None:
                              f'{expected_num_samples}, but got {len(samples)}')
 
         out_dir = os.path.join(args.out_root, split)
-        hashes = get_list_arg(args.hashes)
 
-        if args.progbar:
+        hashes = get_list_arg(args.hashes)
+        if args.progress_bar:
             samples = tqdm(samples, leave=args.leave)
 
-        with MDSWriter(local=out_dir,
+        with MDSWriter(out=out_dir,
                        columns=columns,
                        compression=args.compression,
                        hashes=hashes,
-                       size_limit=args.size_limit) as out:
+                       keep_local=False,
+                       size_limit=args.size_limit,
+                       progress_bar=args.progress_bar) as out:
```

Comment:
if I am reading this correctly, two progress bars?

Reply:
Yes, the two progress bar works in sync and I did ran ade20k example which has two progress bars.